index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
```

Comment:
```suggestion
 * Copyright (c) OpenTofu
```

Much like in "lib/releases.js", should this copyright notice be reworded to OpenTofu as well?

Reply:
I'd like to clarify this bit. All files, but releases.js were copied from the setup-terraform repo. AFAIK, the header shall reflect that.